server/middleware/__tests__/searchHandler.spec.js

```diff
@@ -51,8 +51,8 @@ describe('handleSearch', () => {
                     if (listId === 'S_SYSTEM_CONFIGURATION') {
                         return Promise.resolve({
                             workstackTypeColumns: [
-                                { workstackColumns: {} },
-                                { workstackColumns: {} }
+                                { workstackType: 'DEFAULT', workstackColumns: {} },
+                                { workstackType: 'TypeB', workstackColumns: {} }
```

Comment:
Is the naming `TypeB` arbitrary? If so, a more self-explanatory name might be better: or at least one that matches the CAPITALISED_SNAKE_CASE of everything else. What differentiates Type B from the default? That would be a good starting point for coming up with a more meaningful, but still project independent, name.

Reply:
As you can see, there is no difference between the two test workstack definitions. I just needed one to be marked as 'DEFAULT'. I'm happy to change the other definition to 'Other' it doesn't mean anything.  Are you happy with it or have another suggestion?

Reply:
Ah -- I hadn't noticed this was a test! In which case I guess it's fine, unless you feel like writing a test to check it does actually search `DEFAULT` over `TypeB`. I guess you could go for "Some Other Workstack Type" or something.